packages/edit-site/src/components/global-styles/palette.js

```diff
@@ -55,34 +55,32 @@ function Palette( { name } ) {
 		? '/colors/palette'
 		: '/blocks/' + encodeURIComponent( name ) + '/colors/palette';
 
+	const paletteButtonText =
+		colors.length > 0 ? __( 'Edit palette' ) : __( 'Add colors' );
+
 	return (
 		<VStack spacing={ 3 }>
 			<Subtitle level={ 3 }>{ __( 'Palette' ) }</Subtitle>
 			<ItemGroup isBordered isSeparated>
-				<NavigationButtonAsItem path={ screenPath }>
+				<NavigationButtonAsItem
+					path={ screenPath }
+					aria-label={ paletteButtonText }
+				>
 					<HStack direction="row">
-						{ colors.length > 0 ? (
-							<>
-								<ZStack isLayered={ false } offset={ -8 }>
-									{ colors
-										.slice( 0, 5 )
-										.map( ( { color }, index ) => (
-											<ColorIndicatorWrapper
-												key={ `${ color }-${ index }` }
-											>
-												<ColorIndicator
-													colorValue={ color }
-												/>
-											</ColorIndicatorWrapper>
-										) ) }
-								</ZStack>
-								<FlexItem isBlock>
-									{ __( 'Edit palette' ) }
-								</FlexItem>
```

Comment:
The whole point of https://github.com/WordPress/gutenberg/pull/65124 as discussed in https://github.com/WordPress/gutenberg/issues/65112 is to make the accessible name of this button visible by the means of visible text. The visible label must not be removed. I'm not opposed to design changes as long as this button keeps its visible text.

Reply:
Icon buttons do not have visible test.